.github/workflows/main.yaml

```diff
@@ -39,12 +39,22 @@ jobs:
           git-ref: "${{ github.ref }}"
           nginx-tag: "${{ steps.target_nginx_tag.outputs.tag }}"
           matrix-nginx: "${{ matrix.nginx }}"
-      - uses: docker/build-push-action@v1
+      - uses: docker/setup-qemu-action@v2
+        name: Set up QEMU
+      - uses: docker/setup-buildx-action@v2
+        name: Set up Docker Buildx
+      - uses: docker/login-action@v2
+        name: Login to Docker Hub
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - uses: docker/build-push-action@v4
+        name: Build and push
         if: ${{ steps.target_tags.outputs.tags != '' }}
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: ${{ secrets.DOCKER_REPOSITORY }}
-          tags: ${{ steps.target_tags.outputs.tags }}
-          always_pull: true
-          build_args: NGINX_TAG=${{ steps.target_nginx_tag.outputs.tag }}
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          pull: true
+          tags: ${{ steps.target_image_name.outputs.tags }}
```

Comment:
I think this was a breaking change with `build-push-action@v4`. I guess you need to prefix this tag with the full repository name (`${{ secrets.DOCKER_REPOSITORY }}`). See [these docs](https://docs.docker.com/build/ci/github-actions/local-registry/#:~:text=true%0A%20%20%20%20%20%20%20%20%20%20tags%3A-,localhost%3A5000/name/app%3Alatest,-%2D%0A%20%20%20%20%20%20%20%20name%3A).